lib/stub_ui/lib/painting.dart

```diff
@@ -2669,6 +2670,13 @@ class ImageShader extends Shader {
     super._();
 }
 
+/// A shader (as used by [Paint.shader]) that tiles a picture.
```

Comment:
Please provide additional documentation, e.g. sample code (see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#provide-sample-code).

Reply:
(notably, how to create a PictureShader should be documented here, as well as how to use it)